modules/dqt/.gitignore

```diff
@@ -0,0 +1,13 @@
+js/components
```

Comment:
@maltheism this file and all others inside `modules/dqt/...` should be copied over to the project git repo as an override on the module (the repository name is CBIGR) as well as the webpack content should be moved to webpack-project the entire module can simply be copy pasteed there

Reply:
@ridz1208 I was having an issue getting @HenriRabalais's biobank module to work in the project directory. I was getting eslint error that I couldn't solve. So worried I might run into that with moving DQT to project but I'll try tomorrow.